### PR TITLE
Dashboard Style Cleanup

### PR DESCRIPTION
**Pull request by Fredasaurus**
Makes headers consistent, fixes a button bug is FF.

Before
screen shot 2015-07-28 at 10 58 42 am

After
screen shot 2015-07-28 at 10 59 09 am

Things left to do:

- [ ] Change "Recent Topics" to "Learning Forecast"
- [ ] Just have  "My Weakest Topics" button - move into Learning Forecast panel above "All Topics"
- [ ] Change "Current level of understanding" to "Recent topics" italics, grey, not bold
- [ ] Remove Stronger 
- [ ] Change "X worked" to "X problems worked"

### CODE REVIEW

**Comment by Fredasaurus** [comment]
I had to use !important in  a couple places (!I know), so if someone can figure that one out that would be great.

**Comment by philschatz** [comment]
@nathanstitt any ideas on how to remove the `!important`?

File: resources/styles/components/student-dashboard/progress-guide.less [hunk 1]
```diff
@@ -8,26 +8,42 @@
...
-      .weaker i:after {
-        .fa-icon();
-        padding-left:8px;
-        content: @fa-var-arrow-down;
-        color: @tutor-danger;
+      .weaker {
+        width: 50%;
+        border-top-left-radius: 0 !important;
+        border-bottom-left-radius: 0 !important;
+
```
Comment (nathanstitt) [inline resources/styles/components/student-dashboard/progress-guide.less]: Are you sure the buttons have a border-radius set?  I don't see any rules specifying that and they appear to be in the "flat" style to me.

If they do have a border-radius, the `!important` clause could be dropped if you change the rule to be as specific as whatever applies it.  For instance, here's a case where I had to get pretty specific in order to override some material-design styles: https://github.com/openstax/tutor-js/blob/master/resources/styles/components/student-dashboard/progress-guide.less#L16

**Comment by Fredasaurus** [comment]
The border radius is definitely there. What is strange is that the class "btn-group" should remove it.

Failing other ideas I can up the specificity, thanks.

**Comment by Fredasaurus** [comment]
I also noticed that the box-shadow on buttons isn't a mixin. Should it be?

**Comment by nathanstitt** [comment]
My general rule is that it should be extracted if that set of style are going to be used in more than one place.  I'm not aware of any other use of that so I think it's fine.

**Comment by philschatz** [comment]
@Fredasaurus were you going to remove the remaining `!important` or just leave them in?

**Comment by Fredasaurus** [comment]
Been in meetings all afternoon. Let me try and if I get stuck I'll see if someone can come in in relief.

**Comment by Fredasaurus** [comment]
Looks lie the "btn-group" class is not being applied correctly. Can't figure why.

Here is markup pasted straight from here. http://getbootstrap.com/components/#btn-groups

And how it looks.

screen shot 2015-07-28 at 6 12 35 pm

**Comment by nathanstitt** [comment]
I've noticed that the material design rules change `.btn-group` quite a bit using `not()`  rules.  That's my guess at least

**Comment by Fredasaurus** [comment]
OK added #react-root-container to top of file but.....

I'm assuming that will duplicate this selector when everything compiles,
It seems more block headed that !important.

Let me know, thanks.

**Comment by philschatz** [comment]
Just an idea: what if all CSS rules were prefixed with `#react-root-container`?

**Comment by philschatz** [comment]
This is no longer mergeable. Can you pull from master and resolve conflicts? I think it might be because of Nathan's Learning Guide fixes

**Pull request closed without merging**